stories/object-status/__snapshots__/object-status.stories.storyshot

```diff
@@ -28,20 +28,87 @@ exports[`Storyshots Components/Object Status Clickable Object Status 1`] = `
   </a>
   
 
-  <span
-    class="fd-object-status fd-object-status--critical fd-object-status--link sap-icon--status-critical"
+  <a
+    class="fd-object-status fd-object-status--critical fd-object-status--link"
     role="button"
   >
+    <i
+      class="fd-object-status__icon sap-icon--status-critical"
```

Comment:
These will all need `aria-hidden` or `aria-label` if there is no text

Reply:
This isn't resolved. Please don't mark comments as resolved unless you are the author of the comment.